plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -154,6 +156,26 @@ internal fun Package.toSpdxPackage(
         .filterExcluded()
         .filter(LicenseView.ONLY_DETECTED)
 
+    val declaredPackageLicenses = declaredLicensesProcessed.toSpdxDeclaredLicense()
+    val foundPackageLicenses = declaredPackageLicenses.takeUnless {
+        it == SpdxConstants.NOASSERTION && type == SpdxPackageType.VCS_PACKAGE
+    } ?: run {
+        // If there are of declared licenses and the package type is a VCS, which never have metadata, fall back to
```

Comment:
Why is the logic limited to VCS_PACKAGE?

Reply:
What else should it apply to, `SOURCE_PACKAGE`? These usually come with metadata, so I went for a more strict approach here. But I might reconsider this, please note that this is still a draft.

Reply:
This is obsolete now as the code was removed in the effort to use `licenseInfoResolver` instead of `declaredLicensesProcessed` to access declared licenses.